src/components/elements/MapPolygonPanel/MapMenuPanelItem.tsx

```diff
@@ -62,7 +61,7 @@ const MapMenuPanelItem = ({
     openModal(
       <ModalWithLogo
         uuid={uuid}
-        title={t("Blue Forest")}
+        title={title}
```

Comment:
Is this title inside the ModalWithLogo with transifex?
Otherwise set transifex here.

Reply:
its a name, not needed to be translated.